spec/requests/api/v0/post_manifest_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 describe 'POST manifests' do
+  before(:each) do
+    @current_session = mock_authenticated_session
```

Comment:
inline this? I know it's more code, but I like seeing the setup required for each spec!